dockerfiles/executor_image/Dockerfile

```diff
@@ -39,13 +32,17 @@ RUN DOCKER_VERSION="5:26.1.4-1~debian.12~bookworm" && \
     docker-ce-cli=${DOCKER_VERSION} \
     containerd.io=${CONTAINERD_DEB_VERSION} \
     docker-buildx-plugin=${DOCKER_BUILDX_VERSION} && \
+    apt-get remove -y \
+    curl ca-certificates apt-transport-https && \
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/* && apt-get clean && \
     rm \
-    # keep these in sync with the binaries we are manually adding
+    # keep these in sync with the binaries we are manually
+    # adding in enterprise/server/cmd/executor/BUILD.
+    #
     # rootlesskit
-    /usr/bin/rootlesskit
-    /usr/bin/rootlesskit-docker-proxy
+    /usr/bin/rootlesskit \
+    /usr/bin/rootlesskit-docker-proxy \
```

Comment:
does the fix here need to be backported to the branch that this one is based on?

Reply:
yup, did it.